Jetcaster/wear/src/main/java/com/example/jetcaster/WearApp.kt

```diff
@@ -59,12 +40,8 @@ fun WearApp() {
     val navHostState = rememberSwipeDismissableNavHostState()
     val volumeViewModel: VolumeViewModel = viewModel(factory = VolumeViewModel.Factory)
 
-    // TODO remove from MediaPlayerScaffold
-    val snackBarManager: SnackbarManager = SnackbarManager()
-    val snackbarViewModel: SnackbarViewModel = SnackbarViewModel(snackBarManager)
-
     WearAppTheme {
-        MediaPlayerScaffold(
+        JetcasterScaffold(
```

Comment:
Does it add much to have it in JetcasterScaffold? Should you just inline it here?

Reply:
Good point, removed